src/_locales/en/messages.json

```diff
@@ -15,7 +15,7 @@
 		"message": "Add with release date."
 	},
 	"aboutMessage": {
-		"message": "Sync the history from your favorite streaming services to Trakt.tv!"
+		"message": "Scrobble / sync the history from your favorite streaming services to Trakt.tv!"
```

Comment:
Could we simplify and just say: "Send your activity from your favorite streaming services to Trakt.tv"?

Reply:
Yup